packages/core/src/create/release.ts

```diff
@@ -13,10 +13,10 @@ import { imageSize } from "image-size";
 import type { Keypair, PublicKey } from "@solana/web3.js";
 import type {
   App,
-  Context,
+  Context, MetaplexFileReleaseJsonMetadata,
```

Comment:
NIT: can we put this on its own line, to match the import style?

Reply:
Yes, I think it didn't because I finally turned off auto formatting :)